core/src/consensus/stake/mod.rs

```diff
@@ -383,6 +389,30 @@ pub fn jail(state: &mut TopLevelState, address: &Address, custody_until: u64, ki
     Ok(())
 }
 
+#[allow(dead_code)]
```

Comment:
How about `#[cfg(test)]`?

Reply:
Then dependant code are marked as dead code for non-test build. I don't want propagate #[cfg(test)] to them.